vuln/core/8.json

```diff
@@ -10,5 +10,5 @@
   "affectedEnvironments": [
     "all"
   ],
-  "severity": "unknown"
+  "severity": "critical"
```

Comment:
NVD says this is critical, our blog mentions there is `low` risk... though I wouldn't classify that as Low then... IMO